docs/installation/bootstrap.rst

```diff
@@ -92,6 +94,14 @@ notation for it's various subfields.
       that if you lose the Bootstrap node, you no longer have access to any
       control plane component).
 
+      This ``ip`` for ``ingress`` can be managed by MetalK8s directly if
+      it's possible in your environment, to do so we use
```

Comment:
We may want to explain what "kind of environment" will support this (physical isolated networks, for instance), or at least known limitations (e.g. virtual networks with PortSecurity will not allow arbitrary ARP requests).

Reply:
Right, I will try to add something for this